core/hw/maple/maple_devs.cpp

```diff
@@ -1125,18 +1125,18 @@ struct maple_sega_purupuru : maple_base
 					INC = 0;
 				bool CNT = VIBSET & 1;
 
-				float power = min((POW_POS + POW_NEG) / 7.0, 1.0);
```

Comment:
In general, we have the habit of importing `std` for common functions like `min`. Too lazy to type`std::bs` all the time